windows/rngallery/.gitignore

```diff
@@ -1,2 +1 @@
 /Bundle
-*.pfx

```

Comment:
We'll want to add this line back in. It's a safe guard against publishing pfx files to the repo.